bots/ec2_detach_instance_role.py

```diff
@@ -0,0 +1,38 @@
+'''
+## ec2_detach_instance_role
+What it does: Detach an instance role from an EC2 instance. This role needs be passed in through the params.
+Usage: AUTO: ec2_detach_instance_role
+
+Sample GSL: Instance should have roles

```

Comment:
what about Logic example as well

Reply:
no params needed, so logic example is the same as usage.